packages/core/test-support/helpers/contains-helpers.js

```diff
@@ -0,0 +1,29 @@
+import { findAll } from '@ember/test-helpers';
+
+/**
+ * Finds selector by innerText
+ * @param {String} selector - selector to search
+ * @param {String} text - text to search by
+ * @returns {Element} - first matching dom element
+ */
+export default function findByContains(selector, text) {
+  return findAll(selector).filter(el => el.innerText.includes(text))[0];
```

Comment:
We could just use `find` instead of filter and then we wouldn't need the `[0]` at the end

Reply:
Also what does this return if the element isn't found? Would an ` || null;` at the end be necessary?

Reply:
we need all the dom elements so the filter works.

also it would return undefined which is appropriate I think.

Reply:
But we're only returning the first one that matches the filter which is exactly what find does
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find

Reply:
`return findAll(selector).find(el => el.innerText.includes(text));`